kagglesdk/kaggle_http_client.py

```diff
@@ -173,7 +173,7 @@ def _init_session(self):
       })
 
     self._try_fill_auth()
-    self._fill_xsrf_token(iap_token)
+    # self._fill_xsrf_token(iap_token)
```

Comment:
Is this intentional?

Reply:
Yes, that line causes an error in the MT. This HTTP handler is only used by the API, and I have another, future project to better align auth between it and the base HTTP handler used in `kapigen`. I should have a `TODO` here.